app/js/components/pages/Job.jsx

```diff
@@ -79,19 +78,24 @@ const Job = ({ plugin, action, inputs, submitJob, cancelJob }) => {
                     Cancel
                 </button>
                 <button
+                    disabled={children && children.type.name === 'JobRunning'}
```

Comment:
So it only doesn't work when running a build, I think I know what might be causing it...